mkdocs.yml

```diff
@@ -0,0 +1,121 @@
+site_description: Pantheon Java Ethereum client documentation.
+site_author: Pantheon community
+copyright: Pantheon and its documentation are licensed under Apache 2.0 license /
+  This <a href="https://readthedocs.org/">Readthedocs.org</a> documentation is maintained with love by <a href="https://pegasys.tech/">PegaSys</a>.
+#google_analytics: ['UA-36723568-3', 'pantheon.readthedocs.io']
+
+#extra project info and template customisation
+extra:
+  support:
+    gitter: https://gitter.im/PegaSysEng/pantheon
+    email: support@pegasys.tech
+    footer_layout:
+      background_color: lightseagreen
+      links_color: white
+
+
+# Repository
+repo_name: PegaSysEng/pantheon
+repo_url: https://github.com/PegaSysEng/pantheon
+edit_uri: ""
+
+theme:
+    name: material
+    custom_dir: docs/custom_theme
+    highlightjs: true
+    hljs_languages:
+        - java
+        - bash
+        - bat
+        - toml
+        - json
+        - yaml
+    palette:
+        primary: blue-grey
+        accent: teal
+    favicon: favicon.ico
+    logo:
+      icon: account_balance
+nav:
+    - Installation:
+        - Installation overview: Installation/Overview.md
+        - Install Binary Distribution: Installation/Install-Binaries.md
+        - Build from Source: Installation/Build-From-Source.md
+    - Getting Started:
+        - Getting started overview: Getting-Started/Getting-Started.md
+        - Starting Pantheon: Getting-Started/Starting-Pantheon.md
+        - Running Pantheon from Docker Image: Getting-Started/Run-Docker-Image.md
+        - Private Network Quickstart tutorial: Getting-Started/Private-Network-Quickstart.md
+    - Configuring Pantheon:
+        - Network ID and Chain ID: Configuring-Pantheon/NetworkID-And-ChainID.md
+        - Node Keys: Configuring-Pantheon/Node-Keys.md
+        - Networking: Configuring-Pantheon/Networking.md
+        - Accounts for Testing: Configuring-Pantheon/Accounts-for-Testing.md
+        - Logging: Configuring-Pantheon/Logging.md
+        - Testing and Developing Nodes: Configuring-Pantheon/Testing-Developing-Nodes.md
+        - Proof of Authority: Configuring-Pantheon/Proof-of-Authority.md
+        - Passing JVM Options: Configuring-Pantheon/Passing-JVM-Options.md
+    - Using Pantheon:
+        - Creating and Sending Transactions: Using-Pantheon/Transactions.md
+        - Using Wallets for Account Management: Using-Pantheon/Account-Management.md
+        - Mining: Using-Pantheon/Mining.md
+        - RPC Pub/Sub: Using-Pantheon/RPC-PubSub.md
+        - Debugging Pantheon: Using-Pantheon/Debugging.md
+    - Developement:
+        - Building Pantheon: development/building.md
+        - Code coverage: development/code-coverage.md
+        - Logging:  development/logging.md
+        - Running with Gradle: development/running-with-gradle.md
+    - Contribute:
+        - Code review: community/code-reviews.md
+        - Community membership: community/community-membership.md
+    - References:
```

Comment:
I think typically is would be 'Reference' rather than 'References'.  The plural version makes me imagine I'm going to get a list of references to other information.